src/platforms/esp32/components/avm_builtins/gpio_driver.c

```diff
@@ -18,8 +18,8 @@
  * SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
```

Comment:
I think we should split the GPIO Nif and Port into separate modules, but we can do that in the future.  I have some other cleanup in mind with the API.